frontend/src/pages/TermPlanner/TermPlanner.tsx

```diff
@@ -37,20 +37,15 @@ const TermPlanner = () => {
   const [isDragging, setIsDragging] = useState(false);
 
   const dispatch = useDispatch();
-  const getMarkList = () => Object.values(planner.courses).map(((object) => object.mark));
```

Comment:
got rid of my cringe because chucking the payload into a var made this check much simpler